utility-emissions-channel/typescript_app/tests/test.js

```diff
@@ -64,62 +70,90 @@ describe("Test fabric", function() {
         orgName: "auditor1",
         utilityId: "USA_EIA_11208",
         partyId: "1234567890",
-        fromDate: "2020-04-06T10:10:09Z",
-        thruDate: "2020-04-06T10:10:09Z",
+        fromDate: "2018-05-07T10:10:09Z",
+        thruDate: "2018-05-07T10:10:09Z",
         energyUseAmount: 100,
         energyUseUom: "TONS",
       })
       .end((error, response) => {
         
-        let res = JSON.stringify(response.body.info.includes("Failed to submit transaction"));
-        if(res)
-        {
-        assert(response.status == 409, "Could not reach the gateway network");
+       try{
+        //console.log(`response :  ${JSON.stringify(response.body)}`);
+        expect(response.body.info).to.equal("EMISSION RECORDED TO LEDGER");
         done();
-        }
-        
-        else if(response.body.info == "EMISSION RECORDED TO LEDGER")
-        {
-        assert(response.body.info == "EMISSION RECORDED TO LEDGER", "Emission is successfully recorded.");
-        done();
-        }
+       }
```

Comment:
For the transaction now: if it fails, the tests would fail as well.
They would only pass if we have a response when the transaction is successful.